### PR TITLE
Feature/section11 footer logo

### PR DESCRIPTION
**Pull request by Megan-Sen**

### CODE REVIEW

File: client/src/pages/Home/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+				<div className="home_section11_child1">
+					<img
+						className="footer-logo"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="home_section11_child2">
+					<div>
+						<p>Do want to be part of us?</p>
```
Comment (filkovsp) [inline client/src/pages/Home/Footer/Footer.js:20]: please add `you` in there: `Do want to be part of us?` :)

File: client/src/pages/Home/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
```
Comment (filkovsp) [inline client/src/pages/Home/Footer/Footer.js:1]: do we need this file at all?

File: client/src/pages/Home/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
+
```
Comment (filkovsp) [inline client/src/pages/Home/Footer/Footer.css:1]: do we need this file for this part of the app?

File: client/src/pages/Home/FooterLogo/FooterLogo.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+				<div className="footer-logo-img_container">
+					<img
+						className="footer-logo-img"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="fotter-button_container">
+					<div>
+						<p>Do want to be part of us?</p>
```
Comment (filkovsp) [inline client/src/pages/Home/FooterLogo/FooterLogo.js:18]: please add `you`: 
`Do you want to be part of us?`

**Review by filkovsp — CHANGES_REQUESTED** [review]

File: client/src/pages/Home/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+					<div>
+						<p>Do want to be part of us?</p>
+					</div>
+					<div>
+							<a
+								href="https://example.com/"
+								target="_blank"
+								rel="noreferrer"
+								className="footer-button"
+								onMouseEnter={() => setIsShown(true)}
```
Comment (khaledkz) [inline client/src/pages/Home/Footer/Footer.js:28]: just wonder why you are using onMouseEnter only? do you need yo add onClick? also is href correct

File: client/src/pages/Home/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+					</ul>
+					<ul>
+						<li>List</li>
+						<li>List</li>
+						<li>List</li>
+					</ul>
+				</div>
+				<div className="home_section12_child2">
+					<p>Our Email address</p>
+					<span>info@sensational-life.com</span>
```
Comment (khaledkz) [inline client/src/pages/Home/Footer/Footer.js:59]: can you make the email anchor with mailto: href

File: client/src/pages/Home/FooterLogo/FooterLogo.css [hunk 7]
```diff
@@ -0,0 +1,62 @@
+
+#footerlogo{
+	font-family: 'Ambit-Regular';
+	font-size: 15px;
+    color: var(--main-bg-color);
+	background-color: var(--darkgreen-color);
+    width: 100%;
+}
```
Comment (khaledkz) [inline client/src/pages/Home/FooterLogo/FooterLogo.css:8]: could you please format your code

File: client/src/pages/Home/FooterLogo/FooterLogo.css [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    padding-bottom: 40px;
+    border-bottom: 1px solid var(--main-bg-color);
+}
+
+.footer-logo-img_container{
+    padding: 30px 0px 10px 80px;
+}
+
+.footer-logo-img{
+	width: 250px;
```
Comment (khaledkz) [inline client/src/pages/Home/FooterLogo/FooterLogo.css:23]: same here

File: client/src/pages/Home/FooterLogo/FooterLogo.css [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+.footer-button:hover{
+    background-color: var(--lightorange);
+}
+
+.footer_buttun_tooltip{
+    font-family: 'Ambit SemiBold';
+	font-size: 18px;
+	background-color: var(--main-bg-color);
+	color: var(--darkgreen-color);
```
Comment (khaledkz) [inline client/src/pages/Home/FooterLogo/FooterLogo.css:55]: same here

File: client/src/pages/Home/Footer/Footer.js (hunk 2 above)
Comment (Megan-Sen) [reply client/src/pages/Home/Footer/Footer.js:1]: No, we don't need the following file.
 <Footer folde>
Footer.js
Footer.css  
I need to delete the folder.

File: client/src/pages/Home/Footer/Footer.js (hunk 1 above)
Comment (Megan-Sen) [reply client/src/pages/Home/Footer/Footer.js:20]: No, we don't need the following file.

Footer.js Footer.css I need to delete the folder.

File: client/src/pages/Home/Footer/Footer.css (hunk 3 above)
Comment (Megan-Sen) [reply client/src/pages/Home/Footer/Footer.css:1]: No, we don't need the following file.

Footer.js Footer.css I need to delete the folder.

File: client/src/pages/Home/FooterLogo/FooterLogo.js (hunk 4 above)
Comment (Megan-Sen) [reply client/src/pages/Home/FooterLogo/FooterLogo.js:18]: Oh, thank you, I added it.

File: client/src/pages/Home/Footer/Footer.js (hunk 6 above)
Comment (Megan-Sen) [reply client/src/pages/Home/Footer/Footer.js:59]: I'm sorry that I missed deleting the following file.
Footer.js Footer.css I need to delete the folder.

the email icon is in Section12, so I will add the icon in another component, thank you.

File: client/src/pages/Home/FooterLogo/FooterLogo.css (hunk 7 above)
Comment (Megan-Sen) [reply client/src/pages/Home/FooterLogo/FooterLogo.css:8]: Thank you, I formated.

File: client/src/pages/Home/Footer/Footer.js (hunk 5 above)
Comment (Megan-Sen) [reply client/src/pages/Home/Footer/Footer.js:28]: The instruction in ticket on Trello is " When the User hovers over the Get Started button, the colour of the button changes, and a text box ''Get Started '' appears just below the button.".
There is no instruction for a function when users click the buttun, so I use only onMouseEnter in this component.

**Comment by filkovsp** [comment]
Hi Megan, 
thanks for correcting that. 
I'm ok with the PR now :)

**Pull request merged**